src/main/java/org/example/csv2tex/csv/CsvParsingUtil.java

```diff
@@ -10,4 +10,12 @@ public static String[] splitCompetencyColumnHeader(String columnHeader) {
         // limit 4: will keep all lines after the 4th as part of the 4th split
         return columnHeader.split("(\r\n|\r|\n)", 4);
     }
+
+    public static String checkEmptySetDefaultZero(String absenceTime) {
```

Comment:
Or, just use `org.apache.commons.lang3.StringUtils.defaultString`

Reply:
I'll check that. Thx.